x-pack/plugins/maps/public/actions/layer_actions.ts

```diff
@@ -580,3 +581,23 @@ export function setAreTilesLoaded(layerId: string, areTilesLoaded: boolean) {
     newValue: areTilesLoaded,
   };
 }
+
+export function updateCounts(layerId: string, mbMetaFeatures: Feature[]) {
```

Comment:
"updateCounts" is a very vague name. How about something more descriptive like "updateMetaFromTiles"?